payjoin/src/core/receive/v2/session.rs

```diff
@@ -68,6 +66,21 @@ where
         })?;
     }
 
+    let ctx =
+        history.session_context().expect("Session context should be present after the first event");
```

Comment:
Note: this expect makes more sense after we remove uninitlized as a session state #1014